### ISSUE TITLE
Augur Client 1.1.1: Can't create a new market: Button greyed out and error in console

### ISSUE THREAD

**Issue by SebastianMetzger** [AugurProject/augur-ui#2139]
I tried to create a multi outcome market, on the last page the button is greyed out:
![image](https://user-images.githubusercontent.com/2345498/43686956-1fa9b9ea-98ce-11e8-966a-0e8194436028.png)

When I read the message about gas etc in the console I thought perhaps because I had no ETH in my metamask wallet, but I transfered some and still get the same error.
This is in the console of the UI:
```
inpage.js:1 MetaMask - RPC Error: Error: gas required exceeds allowance or always failing transaction
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:1401895
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:931969
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:350331
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:368562)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353161
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:350331
    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353437)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353475
    at Ht (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:362844)
    at Object.<anonymous> (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353564)
    at e.value (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:932884)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:932451
    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:368506)
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:368584)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353161
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:932417
    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:368506)
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:368584)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353161
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:932790
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353971
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:350331
    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353437)
    at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353359)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353161
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353951
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:932672
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:931109
    at i (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:1070338) {code: -32603, message: "Error: gas required exceeds allowance or always fa…ogaeaoehlefnkodbefgpgknn/background.js:1:1070338)"}code: -32603message: "Error: gas required exceeds allowance or always failing transaction↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:1401895↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:931969↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:350331↵    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:368562)↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353161↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:350331↵    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353437)↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353475↵    at Ht (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:362844)↵    at Object.<anonymous> (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353564)↵    at e.value (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:932884)↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:932451↵    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:368506)↵    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:368584)↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353161↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:932417↵    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:368506)↵    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:368584)↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353161↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:932790↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353971↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:350331↵    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353437)↵    at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353359)↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353161↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:353951↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:932672↵    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:931109↵    at i (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1:1070338)"__proto__: Object
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
(anonymous) @ inpage.js:1
He @ inpage.js:1
(anonymous) @ inpage.js:1
value @ inpage.js:1
(anonymous) @ inpage.js:1
n @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
n @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
s @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
setTimeout (async)
write @ inpage.js:1
b @ inpage.js:1
(anonymous) @ inpage.js:1
g.write @ inpage.js:1
y @ inpage.js:1
(anonymous) @ inpage.js:1
a.emit @ inpage.js:1
w @ inpage.js:1
_ @ inpage.js:1
v.push @ inpage.js:1
_write @ inpage.js:1
b @ inpage.js:1
(anonymous) @ inpage.js:1
g.write @ inpage.js:1
y @ inpage.js:1
(anonymous) @ inpage.js:1
a.emit @ inpage.js:1
w @ inpage.js:1
_ @ inpage.js:1
v.push @ inpage.js:1
i._onMessage @ inpage.js:1
postMessage (async)
c._write @ contentscript.js:1
L @ contentscript.js:1
(anonymous) @ contentscript.js:1
h.write @ contentscript.js:1
I @ contentscript.js:1
(anonymous) @ contentscript.js:1
m.emit @ contentscript.js:1
V @ contentscript.js:1
a @ contentscript.js:1
o.push @ contentscript.js:1
c._onMessage @ contentscript.js:1
EventImpl.dispatchToListener @ VM7743 extensions::event_bindings:403
publicClassPrototype.(anonymous function) @ VM7749 extensions::utils:138
EventImpl.dispatch_ @ VM7743 extensions::event_bindings:387
EventImpl.dispatch @ VM7743 extensions::event_bindings:409
publicClassPrototype.(anonymous function) @ VM7749 extensions::utils:138
dispatchOnMessage @ VM7750 extensions::messaging:392
wrap-on-failed-callback.js:11 transaction failed: Error: Error: gas required exceeds allowance or always failing transaction
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at Ht (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at Object.<anonymous> (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at e.value (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at i (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at Ht (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at Object.<anonymous> (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at e.value (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at i (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at e.exports (parse-ethereum-response.js:8)
    at f (blockchain-message-handler.js:36)
    at l (blockchain-message-handler.js:43)
    at blockchain-message-handler.js:54
    at o.dispatch (index.js:10)
    at Object.<anonymous> (reset-state.js:45)
    at inpage.js:1
    at inpage.js:1
    at i (inpage.js:1)
    at inpage.js:1 {constant: false, name: "createCategoricalMarket", inputs: Array(8), signature: Array(8), returns: "address", …}constant: falseestimateGas: truefrom: "0x92b93bbcb8272e76f0297c9d171294e9252ec02c"gas: "0x5e3918"inputs: (8) ["_endTime", "_feePerEthInWei", "_denominationToken", "_designatedReporterAddress", "_outcomes", "_topic", "_description", "_extraInfo"]name: "createCategoricalMarket"params: (8) ["0x000000000000000000000000000000000000000000000000000000005bb16380", "0x0000000000000000000000000000000000000000000000000011c37937e08000", "0xd5524179cb7ae012f5b642c1d6d700bbaa76b96b", "0x92b93bbcb8272e76f0297c9d171294e9252ec02c", Array(8), "0x43525950544f43555252454e4349455300000000000000000000000000000000", "Bitcoin Price Range 3rd Quarter 2018; September 30th;", "{"longDescription":"The winning contract shall be …om/currencies/bitcoin/","tags":["BITCOIN","BTC"]}"]returns: "address"send: falsesignature: (8) ["uint256", "uint256", "address", "address", "bytes32[]", "bytes32", "string", "string"]to: "0xe991247b78f937d7b69cfc00f1a487a293557677"uuid: "5ea5cf35-2fbb-4e29-abf6-4afd436fd0f0"value: "0x244a2ab6080d40"__proto__: Object undefined 0x28b41c0a4dc236fbe39112d88ad35385cc3461f58b638264d14dde19389ac7dc
(anonymous) @ wrap-on-failed-callback.js:11
(anonymous) @ transact.js:32
e.exports @ parse-ethereum-response.js:8
f @ blockchain-message-handler.js:36
l @ blockchain-message-handler.js:43
(anonymous) @ blockchain-message-handler.js:54
o.dispatch @ index.js:10
(anonymous) @ reset-state.js:45
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
s @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
s @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
(anonymous) @ inpage.js:1
He @ inpage.js:1
(anonymous) @ inpage.js:1
value @ inpage.js:1
(anonymous) @ inpage.js:1
n @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
n @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
s @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
setTimeout (async)
write @ inpage.js:1
b @ inpage.js:1
(anonymous) @ inpage.js:1
g.write @ inpage.js:1
y @ inpage.js:1
(anonymous) @ inpage.js:1
a.emit @ inpage.js:1
w @ inpage.js:1
_ @ inpage.js:1
v.push @ inpage.js:1
_write @ inpage.js:1
b @ inpage.js:1
(anonymous) @ inpage.js:1
g.write @ inpage.js:1
y @ inpage.js:1
(anonymous) @ inpage.js:1
a.emit @ inpage.js:1
w @ inpage.js:1
_ @ inpage.js:1
v.push @ inpage.js:1
i._onMessage @ inpage.js:1
postMessage (async)
c._write @ contentscript.js:1
L @ contentscript.js:1
(anonymous) @ contentscript.js:1
h.write @ contentscript.js:1
I @ contentscript.js:1
(anonymous) @ contentscript.js:1
m.emit @ contentscript.js:1
V @ contentscript.js:1
a @ contentscript.js:1
o.push @ contentscript.js:1
c._onMessage @ contentscript.js:1
EventImpl.dispatchToListener @ VM7743 extensions::event_bindings:403
publicClassPrototype.(anonymous function) @ VM7749 extensions::utils:138
EventImpl.dispatch_ @ VM7743 extensions::event_bindings:387
EventImpl.dispatch @ VM7743 extensions::event_bindings:409
publicClassPrototype.(anonymous function) @ VM7749 extensions::utils:138
dispatchOnMessage @ VM7750 extensions::messaging:392
create-market-form-review.jsx:90 Error: Error: gas required exceeds allowance or always failing transaction
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at Ht (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at Object.<anonymous> (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at e.value (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at i (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at Ht (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at Object.<anonymous> (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at e.value (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at n (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at o (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1
    at i (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background.js:1)
    at e.exports (parse-ethereum-response.js:8)
    at f (blockchain-message-handler.js:36)
    at l (blockchain-message-handler.js:43)
    at blockchain-message-handler.js:54
    at o.dispatch (index.js:10)
    at Object.<anonymous> (reset-state.js:45)
    at inpage.js:1
    at inpage.js:1
    at i (inpage.js:1)
    at inpage.js:1
(anonymous) @ create-market-form-review.jsx:90
onFailed @ estimate-submit-new-market.js:17
(anonymous) @ wrap-on-failed-callback.js:13
(anonymous) @ transact.js:32
e.exports @ parse-ethereum-response.js:8
f @ blockchain-message-handler.js:36
l @ blockchain-message-handler.js:43
(anonymous) @ blockchain-message-handler.js:54
o.dispatch @ index.js:10
(anonymous) @ reset-state.js:45
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
s @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
s @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
(anonymous) @ inpage.js:1
He @ inpage.js:1
(anonymous) @ inpage.js:1
value @ inpage.js:1
(anonymous) @ inpage.js:1
n @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
n @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
s @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
setTimeout (async)
write @ inpage.js:1
b @ inpage.js:1
(anonymous) @ inpage.js:1
g.write @ inpage.js:1
y @ inpage.js:1
(anonymous) @ inpage.js:1
a.emit @ inpage.js:1
w @ inpage.js:1
_ @ inpage.js:1
v.push @ inpage.js:1
_write @ inpage.js:1
b @ inpage.js:1
(anonymous) @ inpage.js:1
g.write @ inpage.js:1
y @ inpage.js:1
(anonymous) @ inpage.js:1
a.emit @ inpage.js:1
w @ inpage.js:1
_ @ inpage.js:1
v.push @ inpage.js:1
i._onMessage @ inpage.js:1
postMessage (async)
c._write @ contentscript.js:1
L @ contentscript.js:1
(anonymous) @ contentscript.js:1
h.write @ contentscript.js:1
I @ contentscript.js:1
(anonymous) @ contentscript.js:1
m.emit @ contentscript.js:1
V @ contentscript.js:1
a @ contentscript.js:1
o.push @ contentscript.js:1
c._onMessage @ contentscript.js:1
EventImpl.dispatchToListener @ VM7743 extensions::event_bindings:403
publicClassPrototype.(anonymous function) @ VM7749 extensions::utils:138
EventImpl.dispatch_ @ VM7743 extensions::event_bindings:387
EventImpl.dispatch @ VM7743 extensions::event_bindings:409
publicClassPrototype.(anonymous function) @ VM7749 extensions::utils:138
dispatchOnMessage @ VM7750 extensions::messaging:392
log-error.js:3 Error instantiating WrappedBigNumber
t.a @ log-error.js:3
a @ create-big-number.js:9
t.e @ format-number.js:205
(anonymous) @ create-market-form-review.jsx:97
(anonymous) @ async.js:726
(anonymous) @ async.js:52
a @ async.js:246
(anonymous) @ async.js:44
(anonymous) @ async.js:723
(anonymous) @ async.js:167
(anonymous) @ get-market-creation-cost-breakdown.js:38
(anonymous) @ bind-contract-function.js:25
(anonymous) @ call-contract-function.js:23
e.exports @ parse-ethereum-response.js:9
f @ blockchain-message-handler.js:36
(anonymous) @ blockchain-message-handler.js:52
o.dispatch @ index.js:10
(anonymous) @ reset-state.js:45
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
s @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
s @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
(anonymous) @ inpage.js:1
He @ inpage.js:1
(anonymous) @ inpage.js:1
value @ inpage.js:1
(anonymous) @ inpage.js:1
n @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
n @ inpage.js:1
i @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
u @ inpage.js:1
s @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
(anonymous) @ inpage.js:1
setTimeout (async)
write @ inpage.js:1
b @ inpage.js:1
(anonymous) @ inpage.js:1
g.write @ inpage.js:1
y @ inpage.js:1
(anonymous) @ inpage.js:1
a.emit @ inpage.js:1
w @ inpage.js:1
_ @ inpage.js:1
v.push @ inpage.js:1
_write @ inpage.js:1
b @ inpage.js:1
(anonymous) @ inpage.js:1
g.write @ inpage.js:1
y @ inpage.js:1
(anonymous) @ inpage.js:1
a.emit @ inpage.js:1
w @ inpage.js:1
_ @ inpage.js:1
v.push @ inpage.js:1
i._onMessage @ inpage.js:1
postMessage (async)
c._write @ contentscript.js:1
L @ contentscript.js:1
(anonymous) @ contentscript.js:1
h.write @ contentscript.js:1
I @ contentscript.js:1
(anonymous) @ contentscript.js:1
m.emit @ contentscript.js:1
V @ contentscript.js:1
a @ contentscript.js:1
o.push @ contentscript.js:1
c._onMessage @ contentscript.js:1
EventImpl.dispatchToListener @ VM7743 extensions::event_bindings:403
publicClassPrototype.(anonymous function) @ VM7749 extensions::utils:138
EventImpl.dispatch_ @ VM7743 extensions::event_bindings:387
EventImpl.dispatch @ VM7743 extensions::event_bindings:409
publicClassPrototype.(anonymous function) @ VM7749 extensions::utils:138
dispatchOnMessage @ VM7750 extensions::messaging:392
log-error.js:4 Error: [BigNumber Error] Not a number: undefined
    at o (bignumber.js:1309)
    at new W (bignumber.js:220)
    at a (create-big-number.js:7)
    at t.e (format-number.js:205)
    at create-market-form-review.jsx:97
    at async.js:726
    at async.js:52
    at a (async.js:246)
    at async.js:44
    at async.js:723
format-number.js:205 Uncaught TypeError: Cannot read property 'times' of undefined
    at t.e (format-number.js:205)
    at create-market-form-review.jsx:97
    at async.js:726
    at async.js:52
    at a (async.js:246)
    at async.js:44
    at async.js:723
    at async.js:167
    at get-market-creation-cost-breakdown.js:38
    at bind-contract-function.js:25
```

**Comment by bthaile** [comment]
https://app.clubhouse.io/augur/story/15020

Looks like an error in the market creation cost estimates. 


Were you adding initial liquidity?

**Comment by SebastianMetzger** [comment]
Initially I did not, but tried that as well. 
At least I added a position in the liquidity stream, but was not prompted to actual transfer anything.
I never before created a market, so I am not sure, if this is normal or not.

**Comment by bthaile** [comment]
Do you have REP? Less than 1 is needed to create a market

**Comment by SebastianMetzger** [comment]
No, I had no REP in my wallet, this is probably the issue. Will try at home in couple of hours if it works.

The UI should communicate this more clearly

**Comment by sharkcrayon** [comment]
I agree, it should.

Adding to issue tracker: https://app.clubhouse.io/augur/story/15030/github-make-sure-message-is-shown-if-user-doesn-t-have-enough-rep-to-create-a-market

**Comment by bthaile** [comment]
We had a bug that stopped the error message from showing. It’ll be fixed in next release.

**Comment by bthaile** [comment]
This is fixed in 1.2.0
